roadmap/1_core-protocol/upcoming/1_canister-groups/canister-groups.md

```diff
@@ -0,0 +1,13 @@
+-
+title: Canister groups
+links:
+  Forum Link: https://forum.dfinity.org/t/canister-groups/16015
+  Proposal: 
+eta:
+is_community: false
+---
+
+Communication between canisters on the same subnetwork is very fast since the messages that they exchange do not need to go through consensus. Since intercanister communication latency is less of a concern, one can scale Dapps through multi-canister architectures, as long as the canisters are guaranteed to be collocated. 
```

Comment:
```suggestion
Communication between canisters on the same subnet is very fast since the messages that they exchange do not need to go through consensus. Since inter canister communication latency is less of a concern, one can scale dapps through multi-canister architectures, as long as the canisters are guaranteed to be collocated. 
```